viewer/static/js/app/components/views/dataset-control.js

```diff
@@ -37,8 +37,13 @@ export default function(props) {
       <div className="dataset-cohorts">
         {props.sortedAllPopulations.map(cohort => {
           const isActive = props.sortedPopulationsToShow.includes(cohort);
+          const numClients = props.currentDataset.populations[cohort].total_clients.toLocaleString('en-US');
+          const numPings = props.currentDataset.populations[cohort].total_pings.toLocaleString('en-US');
           return (
-            <Switch key={cohort} label={cohort} onClick={props.handleCohortSwitch} active={isActive} />
+            <div key={cohort} className="switch-and-counts">
+              <Switch key={cohort} label={cohort} onClick={props.handleCohortSwitch} active={isActive} />
+              <span className="cohort-counts">({numClients} clients / {numPings} pings)</span>
```

Comment:
That's an interesting idea. I'm leaning slightly towards the current approach for now just because it's more discoverable whereas the hover effect takes more effort to notice, but let's see how people react and keep this in mind.